sample/src/main/java/com/springboot/hello/sample/controller/Controller.java

```diff
@@ -0,0 +1,23 @@
+package com.springboot.hello.sample.controller;
+
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+public class Controller {
+	
+	@RequestMapping("/")
+    public String getEmployees() 
+    {
+		return "Hello there";
+    }
+    
+    @RequestMapping("/{username}")
```

Comment:
This is path parameter. Can you change it to query param? Same comment as above for method name as well